DuckDuckGo/NetworkProtectionRootView.swift

```diff
@@ -28,9 +28,8 @@ struct NetworkProtectionRootView: View {
 
     init() {
         let accountManager = AppDependencyProvider.shared.subscriptionManager.accountManager
-        let subscriptionFeatureAvailability = AppDependencyProvider.shared.subscriptionFeatureAvailability
         let locationListRepository = NetworkProtectionLocationListCompositeRepository(accountManager: accountManager)
-        let usesUnifiedFeedbackForm = accountManager.isUserAuthenticated && subscriptionFeatureAvailability.usesUnifiedFeedbackForm
+        let usesUnifiedFeedbackForm = accountManager.isUserAuthenticated // && subscriptionFeatureAvailability.usesUnifiedFeedbackForm
```

Comment:
This is the one open TODO of this PR, the unified feedback form feature flag is now fully rolled out and we shouldn't need to access it any more, but I'll get confirmation of this. In the mean time, the rest of the PR can be reviewed.